service/boltz/boltz.go

```diff
@@ -75,3 +100,135 @@ func (t *BoltzService) GetStatus() (string, error) {
 		return btcStatus.Status + "; " + ltcStatus.Status, nil
 	}
 }
+
+func (t *BoltzService) GetServiceInfo(currency string) (*pb.GetServiceInfoResponse, error) {
+	client, err := t.GetRpcClient(currency)
+	if err != nil {
+		return nil, err
+	}
+	req := pb.GetServiceInfoRequest{}
+	return client.GetServiceInfo(context.Background(), &req)
+}
+
+func (t *BoltzService) Deposit(currency string) (*pb.DepositResponse, error) {
+	client, err := t.GetRpcClient(currency)
+	if err != nil {
+		return nil, err
+	}
+	req := pb.DepositRequest{}
+	req.InboundLiquidity = 10
+	return client.Deposit(context.Background(), &req)
+}
+
+func (t *BoltzService) Withdraw(currency string, amount int64, address string) (*pb.CreateReverseSwapResponse, error) {
+	client, err := t.GetRpcClient(currency)
+	if err != nil {
+		return nil, err
+	}
+	req := pb.CreateReverseSwapRequest{}
+	req.Amount = amount
+	req.Address = address
+	return client.CreateReverseSwap(context.Background(), &req)
+}
+
+func (t *BoltzService) ConfigureRpc(options *RpcOptions) {
+	t.rpcOptions = options
+}
+
+func (t *BoltzService) GetRpcClient(currency string) (pb.BoltzClient, error) {
+	var client pb.BoltzClient
+	var err error
+	if strings.ToLower(currency) == "btc" {
+		client, err = t.GetBtcRpcClient()
+	} else if strings.ToLower(currency) == "ltc" {
+		client, err = t.GetLtcRpcClient()
+	} else {
+		err = errors.New(fmt.Sprintf("Currency %s is not supported", currency))
+	}
+	if err != nil {
+		return nil, err
+	}
+	return client, nil
+}
+
+func (t *BoltzService) CreateRpcClient(port int16) (pb.BoltzClient, error) {
+	addr := fmt.Sprintf("%s:%d", t.rpcOptions.Host, port)
+	var opts []grpc.DialOption
+	opts = append(opts, grpc.WithInsecure())
```

Comment:
@reliveyy I didn't find a TLS cert for Boltz in the .xud-docker folder. Should we create one and use it here to avoid accessing the Boltz RPC in an insecure way or is the xud-docker environment safe and closed enough to leave it as is?